packages/app/src/systems/Home/routes.tsx

```diff
@@ -3,18 +3,28 @@ import { Outlet, Route } from 'react-router-dom';
 import { Pages } from '../Core/types';
 import { FaucetDialog } from '../Faucet/components';
 
-import { Home } from './pages';
+import { Home, Receive } from './pages';
 
 export const homeRoutes = (
-  <Route
-    path={Pages.wallet()}
-    element={
-      <>
-        <Home />
-        <Outlet />
-      </>
-    }
-  >
+  <Route path={Pages.wallet()}>
+    <Route
+      index
+      element={
+        <>
+          <Home />
+          <Outlet />
+        </>
+      }
+    />
     <Route path={Pages.faucet()} element={<FaucetDialog />} />
+    <Route
+      path={Pages.receive()}
+      element={
+        <>
+          <Receive />
+          <Outlet />
```

Comment:
do we need to include `<Outlet />` here? `Home` had it because we might need go to Faucet route